code/ObjFileParser.cpp

```diff
@@ -182,9 +182,10 @@ void ObjFileParser::parseFile()
 
         case 'm': // Parse a material library or merging group ('mg')
             {
-				std::string name;
+                std::string name;
 
-				getName(m_DataIt, m_DataItEnd, name);
+                getName(m_DataIt, m_DataItEnd, name);
+                name = name.substr(0, name.find(" "));
```

Comment:
Thanks for reviewing my code.

If I understand correctly, `name.find(" ")` will return `npos` if `name` doesn't contain a space, and `name.substr(0, npos)` [should `== name`](http://en.cppreference.com/w/cpp/string/basic_string/substr). So if `name` contains no spaces, won't the extra line have no effect?